app/controllers/books_controller.rb

```diff
@@ -0,0 +1,6 @@
+class BooksController < ApplicationController
+
+    def show
+    end
+    
+end
```

Comment:
より望ましいだけなので、参考までに 🙆‍♂️ 
```suggestion
class BooksController < ApplicationController
  def show
  end
end
```